packages/client/shared/gqlIds/JiraDimensionFieldId.ts

```diff
@@ -0,0 +1,10 @@
+
+const JiraDimensionFieldId = {
+  join: (cloudId: string, dimensionName: string, fieldId: string) => `${cloudId}:${dimensionName.replace(/\s/g, '_')}:${fieldId}`,
+  split: (id: string) => {
+    const [cloudId, dimensionName, fieldId] = id.split(':')
+    return {cloudId, dimensionName: dimensionName.replace(/_/g, ' '), fieldId}
```

Comment:
you're right. url-safe base64 would be superior, but the server& client have different functions to convert to base64, so i just did this. we could 

- disable underscores in dimension names (not great for users)
- hope people don't use underscores in dimension names (current approach :see_no_evil: )
- download a new base64 package that works on server & client (bloat & maintenance :cry: )
- create a new helper to be used just on the server (kinda defeats the whole purpose because we want to easily tell which files depend on joining & splitting the id)
- ???

any recommendations?

Reply:
😂 Fun with me